packages/elements/src/datetime-picker/index.ts

```diff
@@ -537,6 +524,19 @@ export class DatetimePicker extends ControlElement implements MultiValue {
     this.addEventListener('tap', this.onTap);
   }
 
+  /**
+   * Perform asynchronous update
+   * @returns promise
+   */
+  protected override async performUpdate(): Promise<void> {
+    [this.toAriaLabel, this.fromAriaLabel] = await Promise.all([
+      this.labelTPromise(DateTimePickerDataName.to.toUpperCase()),
+      this.labelTPromise(DateTimePickerDataName.from.toUpperCase())
```

Comment:
with update to `DateTimePickerDataName`, there should be need to call `toUpperCase()` now.

Reply:
Fixed